nimbus-features/onboardingFeature.yaml

```diff
@@ -12,14 +12,32 @@ features:
           If true, we show the new Onboarding screen when the user for v106 version.
         type: Boolean
         default: true
+      notification-card-position:
+        description: >
+          This provides the setting for how we show the notification card in onboarding.
+        type: NotificationCard
+        default: no-card
     defaults:
       - channel: beta
         value: {
           "upgrade-flow": false,
-          "first-run-flow": true
+          "first-run-flow": true,
+          "notification-card-position": no-card
         }
       - channel: developer
         value: {
           "upgrade-flow": false,
-          "first-run-flow": true
+          "first-run-flow": true,
+          "notification-card-position": no-card
```

Comment:
should the dev default be `no-card`? Do we want to see something, at least? I have no deep thoughts re: this. Just asking.

Reply:
Yeah, we could definitely have something else as default for devs. Good idea.